reinforcement_learning/Lesson-3-Value-Based-Learning-Method-draft.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>Corrected typo: "I will get a..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>